docs/workflow/requirements/windows-requirements.md

```diff
@@ -108,3 +108,33 @@ You can also temporarily add a directory to the PATH environment variable with t
 You can make your change to the PATH variable persistent by going to _Control Panel -> System And Security -> System -> Advanced system settings -> Environment Variables_, and select the `Path` variable under `System Variables` (if you want to change it for all users) or `User Variables` (if you only want to change it for the current user).
 
 Simply edit the PATH variable's value and add the directory (with a semicolon separator).
+
+### Windows on Arm64
+
+The Windows on Arm64 development experience has improved substantially over the last few years, however there are still a few steps you should take to improve performance when developing runtime libraries on an ARM device.
+
+Runtime libraries source their compilers from the [Microsoft.NET.Compilers.Toolset](https://www.nuget.org/packages/Microsoft.Net.Compilers.Toolset/) package which currently doesn't bundle native Arm64 bits. This can result in [suboptimal performance](https://github.com/dotnet/runtime/issues/104548) when working on libraries from Visual Studio. The issue can be worked around by [configuring the registry](https://github.com/dotnet/runtime/issues/104548#issuecomment-2214581797) to run the compiler bits as Arm64 processes.
```

Comment:
```suggestion
The repo build use compilers from the [Microsoft.NET.Compilers.Toolset](https://www.nuget.org/packages/Microsoft.Net.Compilers.Toolset/) package which currently doesn't bundle native Arm64 bits for .NET Framework. This can result in [suboptimal performance](https://github.com/dotnet/runtime/issues/104548) when working on libraries from Visual Studio. The issue can be worked around by [configuring the registry](https://github.com/dotnet/runtime/issues/104548#issuecomment-2214581797) to run the compiler bits as Arm64 processes. The proper fix that will make this workaround unnecessary is being worked on in https://github.com/dotnet/roslyn/pull/74285. 
```